### PR TITLE
Transactions limit

### PR DESCRIPTION
**Pull request by EduBeghe**
- Fixed Header
- Added limit to transactions fetch from Plaid, and loop to fetch all available transactions

### CODE REVIEW

File: frontend/src/app/core/header/header.component.html [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <div style="height:90px;background:#141E28;color:#fff">
     <div style="margin:auto;width:80%">
-        <div style="float:left">
+        <div class="float-left header-container">
             <a routerLink="/dashboard">
-                <img src="assets/images/green-growth-logo.png" style="height:40px;margin-top:22px">
```
Comment (MichaelAshmead) [inline frontend/src/app/core/header/header.component.html:5]: Also, remove the asset from the assets folder

File: backend/routes/index.ts [hunk 2]
```diff
@@ -61,27 +59,42 @@ router.get('/transactions', decodeIDToken, checkJwt, async function (req, res, n
...
+    let actualCount = options.offset;
+    let totalTransactions = options.count; //Just to enter the loop
     const businessLocationsForBusiness = await queries.getBusinessLocation(email);
     const defaultBusinessLocationId = businessLocationsForBusiness.businessLocationID; // Temporarily default to user's first business location
-    const transactions = await queries.getTransactions(defaultBusinessLocationId);
 
-    for (const transaction of transactionsResponse.transactions) {
-      const foundTransaction = _.find(transactions, { 'amount': transaction.amount, 'date': transaction.date, 'name': transaction.name });
+    while(actualCount < totalTransactions) {
+      var start = new Date();
```
Comment (MichaelAshmead) [inline backend/routes/index.ts:74]: I don't think this variable is being used, so remove

File: frontend/src/app/core/header/header.component.html (hunk 1 above)
Comment (EduBeghe) [reply frontend/src/app/core/header/header.component.html:5]: Done

File: backend/routes/index.ts (hunk 2 above)
Comment (EduBeghe) [reply backend/routes/index.ts:74]: Done

**Pull request merged**